mne/viz/_3d.py

```diff
@@ -1849,6 +1865,8 @@ def _check_time_viewer_compatibility(time_viewer, show_traces):
         if not check_version('surfer', '0.9'):
             raise RuntimeError('This function requires pysurfer version '
                                '>= 0.9')
+    if time_viewer:
+        time_viewer = _check_stc_time(stc, hemi)
```

Comment:
We don't want to always disable it, it controls more than time so can still be useful even if there is a single time point